### PR TITLE
Update syncthru for config flow

### PR DESCRIPTION
**Pull request by scop**
## Proposed change

Syncthru config entry docs, for https://github.com/home-assistant/core/pull/36690

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36690
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/syncthru.markdown [hunk 1]
```diff
@@ -13,30 +14,12 @@ ha_domain: syncthru
...
-    required: false
-    default: Samsung Printer
-    type: string
-{% endconfiguration %}
+
+### Setup
+
+Go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Samsung SyncThru Printer**.
+
+If your printer is on and you have enabled [SSDP](/../ssdp) discovery, it's likely that the printer is automatically discovered. In that case, you shouldn't need any information to complete configuration - simply confirm that the information displayed is correct.
```
Comment (klaasnicolaas) [inline source/_integrations/syncthru.markdown]:
```suggestion
If your printer is on and you have enabled [SSDP](/integrations/ssdp) discovery, it's likely that the printer is automatically discovered. In that case, you shouldn't need any information to complete configuration - simply confirm that the information displayed is correct.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/syncthru.markdown (hunk 1 above)
Comment (scop) [reply source/_integrations/syncthru.markdown]: Good catch, thanks. I opted to change it to `../ssdp/` instead, as hardcoding `integrations` does not seem to serve a purpose here.

Comment (frenck) [reply source/_integrations/syncthru.markdown]: Actually, we should not do that @scop. I'm committing the suggestion. Using a standard is more important if we ever change the structure.

**Review by frenck — APPROVED** [review]

File: source/_integrations/syncthru.markdown (hunk 1 above)
Comment (scop) [reply source/_integrations/syncthru.markdown]: @frenck if you prefer it that way:
```
$ git grep -F ../ source/_integrations/
source/_integrations/huawei_lte.markdown:or using YAML. Additionally, if the [SSDP integration](../ssdp/) is
source/_integrations/ssdp.markdown: - [deCONZ](../deconz/)
source/_integrations/ssdp.markdown: - [Huawei LTE](../huawei_lte/)
source/_integrations/ssdp.markdown: - [Philips Hue](../hue/)
source/_integrations/ssdp.markdown: - [Samsung TV](../samsungtv/)
source/_integrations/ssdp.markdown: - [UPnP](../upnp/)
source/_integrations/ssdp.markdown: - [Sony Songpal](../songpal/)
source/_integrations/vallox.markdown:For convenient switching of ventilation profiles in the GUI, consider using an [input_select](../input_select) hooked to an automation, for example:
```

**Pull request merged**